docs/quick-start/README.md

```diff
@@ -22,8 +22,12 @@ Dragonfly DB will answer to both `http` and `redis` requests out of the box!
 
 You can use `redis-cli` to connect to `localhost:6379` or open a browser and visit `http://localhost:6379`
 
-**NOTE**: On some configurations, running with the `docker run --privileged ...` flag can fix some
-initialization errors.
+**NOTES**:
+
+- On some configurations, running with the `docker run --privileged ...` flag can fix some
+  initialization errors.
+- The `network=host` option doesn't work well on macOS. You can remove it and replace it with `-p 6379:6379` for
+  ports to be successfully mapped between the host and the container.
```

Comment:
I also considered having multiple sections above, something like:

### On linux

```bash
docker run --network=host --ulimit memlock=-1 docker.dragonflydb.io/dragonflydb/dragonfly
```

### On macOS

```bash
docker run --p 6379:6379 --ulimit memlock=-1 docker.dragonflydb.io/dragonflydb/dragonfly
```

Reply:
This option looks even better. Whatever you prefer.

Reply:
Agreed, just changed it.